### PR TITLE
Fix deadlink | Add some packets

### LINKED ISSUE #186: shop_close
shop_close missing

### LINKED ISSUE #195: ta_call missing
ta_call packet is missing

### LINKED ISSUE #196: bsc missing
bsc packet is missing

### LINKED ISSUE #229: ascr missing
ascr packet is missing

### PR DESCRIPTION
**Pull request by Erwann9875**
- Fix #196 
- Fix #195 
- Fix #186 
- Fix #229

### CODE REVIEW

File: src/NosCore.Packets/ServerPackets/UI/ShopClosePacket.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+using NosCore.Packets.Attributes;
+using NosCore.Packets.Enumerations;
+
+namespace NosCore.Packets.ServerPackets.UI
+{
+    [PacketHeader("shopclose", Scope.InGame)]
+    public class ShopClosePacket : PacketBase
+    {
+        [PacketIndex(0, IsOptional = true)]
```
Comment (0Lucifer0) [inline src/NosCore.Packets/ServerPackets/UI/ShopClosePacket.cs]: No need for optional but need byte?

File: src/NosCore.Packets/ServerPackets/Arena/BscPacket.cs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+using NosCore.Packets.Attributes;
+using NosCore.Packets.Enumerations;
+
+namespace NosCore.Packets.ServerPackets.Arena
+{
+    [PacketHeader("bsc", Scope.InGame)]
+    public class BscPacket : PacketBase
+    {
+        [PacketIndex(0)]
+        public byte Type { get; set; }
```
Comment (0Lucifer0) [inline src/NosCore.Packets/ServerPackets/Arena/BscPacket.cs:16]: Sound like en un to me

Comment (Erwann9875) [reply src/NosCore.Packets/ServerPackets/Arena/BscPacket.cs:16]: I think too but dunno which value can be take by this packet need to check

File: src/NosCore.Packets/ServerPackets/UI/AscrPacket.cs [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        public int ArenaTc { get; set; }
+
+        [PacketIndex(6)]
+        public int KillGroup { get; set; }
+
+        [PacketIndex(7)]
+        public int DieGroup { get; set; }
+
+        [PacketIndex(8)]
+        public AscrPacket? Type { get; set; }
```
Comment (0Lucifer0) [inline src/NosCore.Packets/ServerPackets/UI/AscrPacket.cs]: ?

Comment (Erwann9875) [reply src/NosCore.Packets/ServerPackets/UI/AscrPacket.cs]: Ahahaha, my bad

**Pull request merged**